frontend/routes/onboarding/import-wallet/import-wallet.spec.tsx

```diff
@@ -37,6 +37,22 @@ describe('ImportWallet', () => {
     expect(screen.getByTestId(importMnemonicSubmit)).toHaveTextContent('Import wallet')
   })
 
+  it('renders loading state while wallet is being imported', async () => {
+    // 1101-BWAL-078 - I can see the button is disabled and a loading state after submitting
+    mockClient()
+    renderComponent()
+    fireEvent.change(screen.getByTestId(importMnemonic), {
+      target: {
+        value:
+          'solid length discover gun swear nose artwork unfair vacuum canvas push hybrid owner wasp arrest mixed oak miss cage scatter tree harsh critic believe'
+      }
+    })
+    fireEvent.click(screen.getByTestId(importMnemonicSubmit))
+
+    await waitFor(() => expect(screen.getByTestId(importMnemonicSubmit)).toHaveTextContent('Importing…'))
```

Comment:
absolute nitpick but I think if the create loading message is 'Creating wallet...' then this should be 'Importing wallet...'